### DISCUSSION TITLE
Can ART use defense on object detector or just can be used on classifier ?

### DISCUSSION THREAD

**Discussion by Hazzng** [Trusted-AI/adversarial-robustness-toolbox#1903] [Q&A]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by beat-buesser** [comment]
Hi @QuangNguyen2609 Preprocessing "defences" can be applied with all estimators including classifiers and object detectors.
